core/src/main/java/io/grpc/internal/AbstractManagedChannelImplBuilder.java

```diff
@@ -124,6 +124,10 @@
   private boolean enableStatsTagPropagation;
   private boolean enableTracing;
 
+  /**
+   * Sets the maximum message size allowed for a single gRPC frame. Any messages larger than this
+   * limit will result in a {@link io.grpc.StatusRuntimeException} of type RESOURCE_EXHAUSTED.
```

Comment:
StatusRuntimeException is actually an artifact of the stubs, so I'd rather not say it here.

How about:
If an inbound messages larger than this limit is received it will not be processed and the RPC will fail with RESOURCE_EXHAUSTED.